tests/zfs-tests/tests/functional/quota/quota.kshlib

```diff
@@ -62,7 +62,7 @@ function fill_quota
 
 	typeset -i file_size=`$LS -ls $MNTPT/$TESTFILE1 | $AWK '{ print $1 }'`
 	typeset -i limit=0
-	(( file_size = file_size * 512 ))
+	(( file_size = file_size * $BLOCK_SIZE ))
```

Comment:
thank you for your review, i will add the -k option force the block size to 1k for illumos and Linux, and run to check that all of these tests do pass.